app.py

```diff
@@ -0,0 +1,95 @@
+from flask import Flask, request, send_file
+from flask_limiter import Limiter
+from PIL import Image, ImageOps
```

Comment:
Can you separate this file?
app.py for the server and imgsomething.py for dealing with imgs?

Reply:
Done!